### ISSUE TITLE
DNS setup for 100Automations.com/.org/.net

### ISSUE THREAD

**Issue by NivenPrasad** [100Automations/Website#4]
### Overview
Figure out how to point the DNS at GitHub url.

### Action Items

- [ ] Review current setup for HackforLA.org
- [ ] Identify exact settings 
- [ ] Bonnie log in to Google Domains and change DNS

### Resources/Instructions
Cloudflare login

**Comment by jafow** [comment]
not sure if we're using github pages to host a static site, but if so configuring the DNS for a github page was surprisingly simple last time I did it. [here's a link to the relevant docs](https://docs.github.com/en/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site)

**Comment by ExperimentsInHonesty** [comment]
Yep. Ghpages to host a static site.  We use the same method on hackforla.org

**Comment by ExperimentsInHonesty** [comment]
Have currently forwarded the traffic to https://github.com/100Automations and will point the dns at the website url: 100automations.github.io

**Comment by NivenPrasad** [comment]
Moving this issue back to IceBox until website is ready to be live at the URL.

**Comment by ExperimentsInHonesty** [comment]
I reset the settings.  Check at next meeting

**Comment by ExperimentsInHonesty** [comment]
DNS is now resolving to the readme properly again.
